### PR TITLE
Checking and Fixing URLs in the data

### PR DESCRIPTION
**Pull request by khandelwal**
Our contact dataset contained a number of URLs about where the FOIA-centric webpage for that agency/department and the FOIA request form URL. 

Unfortunately a number of those webpages didn't work (returning 404 errors, or worse). 

This pull request contains a script to run the URL checker (we'll likely need to do this periodically). It also contains fixes for almost all the broken URLs.

### CODE REVIEW

File: contacts/data/FCA.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ departments:
   name: Farm Credit Administration
   phone: 703-883-4431
   public_liaison: 'Christine D. Quinn, Phone: (703) 883-4108'
-  request_form: http://www.fca.gov/OGC/FOIARequ.nsf/FOIARequest.xsp
+  request_form: ''
```
Comment (konklone) [inline contacts/data/FCA.yaml:15]: Should this be an empty string?

Comment (khandelwal) [inline contacts/data/FCA.yaml:15]: Yeah, it should actually. They no longer have an online request form. We don't have a method to 'delete' data manually, so I set it to empty string. If there's a better YAML way to do that, let me know.

**Comment by konklone** [comment]
This is such a great batch of data cleaning! Wow.

Does the README need an update to explain any changes to the data updating process?

**Comment by khandelwal** [comment]
No, the README doesn't need a change right now. I know I added in a new 'layer with manual data' script - but it's not currently intended to be part of the pipeline quite yet. It was useful for pushing through this set of updates. We'll have to see if it's useful more longterm to be adhoc.

**Comment by konklone** [comment]
Then sounds good to me! Thanks for slogging through this, @khandelwa.

**Pull request merged**